_concepts/financial-assistance-award-summary.md

```diff
@@ -30,10 +30,13 @@ tabs:
     archived_text:
     description: "Key Features:"
     features:
-      - description: Lorum ipsum
-    disclaimer_text: <a class="usa-cta external-link" href='http://9bgx80.axshare.com/...' target="_blank">View an interactive version of the below image</a>
-    image: "/assets/img/Contract-Award-Summary_06.30.2016.jpg"
-    image_alt_text: This image displays 
+      - description: Details for prime recipients
+      - description: Breakdown of sub-award recipients
+      - description: Visualization of funding
+      - description: Visual timeline of the financial assistance activity
+    disclaimer_text: <a class="usa-cta external-link" href='http://http://http://2m12ll.axshare.com/financial_assistance_award_summary.html' target="_blank">View an interactive version of the below image</a>
+    image: "/assets/img/financial-assistance-summary.jpg"
```

Comment:
I think the image is actually at `/assets/img/concept_financial-assistance-summary.jpg`